package.json

```diff
@@ -47,8 +47,8 @@
 		"fixed-width"
 	],
 	"dependencies": {
+		"eastasianwidth": "^0.2.0",
```

Comment:
Seems like this one has not been updated for many years and it has some bugs.

Reply:
This uses the same method as  [power-assert](https://github.com/twada/power-assert-runtime/pull/14#issuecomment-230982557).

As far as I can see, the current issue is fine.
https://github.com/komagata/eastasianwidth/issues/7#issuecomment-1003531848

Reply:
👍